File: LeetCode/1455/solution.cpp
//  1455. Check If a Word Occurs As a Prefix of Any Word in a Sentence
class Solution {
public:
    int isPrefixOfWord(string sentence, string searchWord) {
        stringstream ss(sentence);
        string word;
        int idx = 1;

        while(ss >> word){
            if(!word.find(searchWord)) return idx;
            idx++;
        }

        return -1;
    }
};

File: LeetCode/1455/solution.py
# 1455. Check If a Word Occurs As a Prefix of Any Word in a Sentence
class Solution:
    def isPrefixOfWord(self, sentence: str, searchWord: str) -> int:
        words = sentence.split(' ')
        for i, word in enumerate(words, 1):
            if word.startswith(searchWord):
                return i
        return -1;
        